lib/mindwendel_web/live/brainstorming_live/show.ex

```diff
@@ -85,4 +85,8 @@ defmodule MindwendelWeb.BrainstormingLive.Show do
   def handle_event("sort", %{"id" => id}, socket) do
     {:noreply, assign(socket, :ideas, Brainstormings.list_ideas_for_brainstorming(id))}
   end
+
+  def handle_event("sort_by_label", %{}, socket) do
+    {:noreply, assign(socket, :ideas, Enum.sort_by(socket.assigns.ideas, & &1.label, :asc))}
```

Comment:
this behaves rather differently, than the sort by likes count. Here we do not reload data from the database, while we do that with the like count sorting. I would propose to query the db in both cases, it also reduces the need for sorting functions in the view.

Reply:
Done. Aligned with other sort method.